<!DOCTYPE html>
<html lang="en" dir="ltr">



<head>

  <meta charset="utf-8">
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <title>Ambar's CV</title>

  <!-- Bootstrap CSS -->
  <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.2/css/bootstrap.min.css"
    integrity="sha384-JcKb8q3iqJ61gNV9KGb8thSsNjpSL0n8PARn9HuZOnIxN0hoP+VmmDGMN5t9UJ0Z" crossorigin="anonymous">

  <!--from google font-->
  <link href="https://fonts.googleapis.com/css2?family=Ubuntu&display=swap" rel="stylesheet">

  <!--fontawesome code added-->
  <script src="https://kit.fontawesome.com/4fab1a568a.js" crossorigin="anonymous"></script>

  <!-- <link rel="stylesheet" href="css/styles.css"> -->
  <link rel="stylesheet" href="css/styles.css">

  <!-- Optional JavaScript -->
  <!-- jQuery first, then Popper.js, then Bootstrap JS -->
  <script src="https://code.jquery.com/jquery-3.5.1.slim.min.js"
    integrity="sha384-DfXdz2htPH0lsSSs5nCTpuj/zy4C+OGpamoFVy38MVBnE+IbbVYUew+OrCXaRkfj" crossorigin="anonymous">
  </script>
  <script src="https://cdn.jsdelivr.net/npm/popper.js@1.16.1/dist/umd/popper.min.js"
    integrity="sha384-9/reFTGAW83EW2RDu2S0VKaIzap3H66lZH81PoYlFhbGU+6BZp6G7niu735Sk7lN" crossorigin="anonymous">
  </script>
  <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.5.2/js/bootstrap.min.js"
    integrity="sha384-B4gt1jrGC7Jh4AgTPSdUtOBvfO8shuf57BaghqFfPlYxofvL8/KUEfYiJOMMV+rV" crossorigin="anonymous">
  </script>

  <!-- details for adding favicon -->
  <link rel="shortcut icon" href="favicon.ico" type="image/x-icon">

  <meta name="description" content="All about personal and research activities of Ambarnil Ghosh">
  <meta name="keywords" content="bioinformatics, computational biology, immunology, protein, DNA, virus,
     python, R, drawing, compuationl drug design, peptide">
  <meta name="author" content="Ambarnil Ghosh">

</head>







<body>


  <section id="nvabarBlock">


    <!--this whole section is for navbar construction-->
    <nav class="navbar bg-transparent navbar-expand-lg navbar-light" style="padding: 3% 3%;">

      <!--this section is the hamburger menu and linked to below content using the id navbarAmbarwebpage01-->
      <a class="navbar-brand" href="index.html">AMBARNIL GHOSH</a>
      <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarAmbarwebpage01"
        aria-controls="navbarAmbarwebpage01" aria-expanded="false" aria-label="Toggle navigation">
        <span class="navbar-toggler-icon"></span>
      </button>


      <!--this section is made for the content of hamburger menu and linked with the id navbarAmbarwebpage01-->
      <div class="collapse navbar-collapse" id="navbarAmbarwebpage01">
        <ul class="navbar-nav ml-auto">
          <li class="nav-item">
            <a class="nav-link" href="index.html">HOME</a>
          </li>
          <li class="nav-item">
            <a class="nav-link" href="research.html">RESEARCH</a>
          </li>
          <li class="nav-item">
            <a class="nav-link" href="CV.html">CV</a>
          </li>
          <li class="nav-item">
            <a class="nav-link" href="myhobbies.html">HOBBIES</a>
          </li>
          <li class="nav-item">
            <a class="nav-link" href="myblog.html">BLOG</a>
          </li>
          <li class="nav-item">
            <a class="nav-link" href="contactme.html">CONTACT</a>
          </li>
        </ul>
      </div>

      <!--this is the searchbar within navbar  
          <form class="form-inline">
            <input class="form-control mr-sm-2" type="search" placeholder="Search" aria-label="Search">
            <button class="btn btn-outline-dark my-2 my-sm-0" type="submit">Search</button>
          </form>
          -->

    </nav>
    <!--it's the end of navbar-->

  </section>




  <section id="acadPositionsBlock">

    <div class="container-fluid">

      <h2>Academic Positions</h2>

      <table>

        <tr>
          <th style="font-size: larger;">Positions</th>
          <th style="font-size: larger;">Details</th>
          <th style="font-size: larger;">Years</th>          
        </tr>

        
        <tr>
          <td>
            Research Fellow at <strong><a href="https://people.ucd.ie/ambarnil.ghosh">University College Dublin</a></strong> (Current Position).
          </td>
          <td>I joined UCD in 2019 as a <b>Marie Skłodowska-Curie Caree-FIT Fellow</b> and my fellowship is funded by European Union’s Horizon2020 
            research and innovation programme under Marie Skłodowska-Curie Caree-FIT grant. The area of my research 
            project is: characterising proteolysis and developing software/webserver for the analysis of proteolysis 
            data obtained from food digestion. In this project, as an industrial secondment, I am currently working with 
            Carbery Group, Cork, Ireland.
          </td>
          <td>
            2019 to current.
          </td>
        </tr>



        <tr>
          <td>
            Research Fellow at <strong><a href="https://www.a-star.edu.sg/imcb A*STAR Singapore">Institute of Molecular and Cell Biology</a></strong> A*STAR Singapore.
          </td>
          <td>
            I was involved with NGS data analysis for various projects. Expression of a specific set of long non-coding
            RNAs and their influence on molecular regulation of inflammation was one of them. NFkB orchestrated regulation
            of various inflammatory pathways were also included in the area of work. Beside working with the academic 
            workflow, I was also involved with PuRPOSE program, which was an industry-academic collaborative platform 
            for providing precision medicine service to liver cancer patients.
          </td>
          <td>
            2018
          </td>
        </tr>



        <tr>
          <td>
            Research Fellow at <strong><a href="https://aim.ibs.re.kr/html/aim_en/">Academy of Immunology and Microbiology</a></strong> POSTECH South Korea.
          </td>
          <td>
            I did computational genomics to understand the relation between immunity and immunomodulatory components. I was also involved in
             performing comparative genomics and metagenonmics analysis to investigate various aspect of probiotics. Key areas & expertise earned 
             from this job: NGS data analysis, Metagenomics analysis, T-cell repertoire analysis, Regulatory genomics mining tools,
             Genome annotation.          
            </td>
          <td>
            2017 to 2018
          </td>
        </tr>





        <tr>
          <td>
            Postdoctoral Researcher (Research Professor) at <strong><a href="http://www.skkumed.ac.kr/">School of Medicine</a></strong> Sungkyunkwan University, South Korea.
          </td>
          <td>
            I did docking, virtual screening and molecular dynamics simulation for in silico small-molecule and peptides based drug discovery.
             I was also involved in doing motif prediction, cancer genomics and NGS data analysis to solve interesting research problems related
             to the role of regulatory mutations in cancer genomics. Performed extensive communication and collaboration with more than five
             experimental research groups. Actively participated in design and implementation of theories/predictions for experimental projects.
             Involved in handling super-computer node at DGIST (Daegu Gyeongbuk Institute of Science and Technology) and CUDA-core based molecular
             dynamics simulation in GROMACS. Role of account administrator of Sungkyunkwan University for the NCBI eRA-dbGAP account including 
             TCGA/GDC cancer genome database.Extensive shell scripting and Python scripting for handling regex based prediction of motifs and 
             other genomics pipelines. Key areas & expertise learned are: Protein & DNA motif prediction, MD simulation (GROMACS), 
             Protein homology modeling, Cancer genomics (patient sample data), Shell Scripting, Python, Software used: Discovery Studio,
             MOE. OS: Ubuntu 12.04 LTS        
            </td>
          <td>
            2014 to 2017
          </td>
        </tr>





        <tr>
          <td>
            Research Associate at <strong><a href="http://www.saha.ac.in/web/npd-home">C&MB Division SINP</a></strong>
             , Kolkata, India.
          </td>
          <td>
            I did comparative genomics, druggable genome mining, pharmacophore modelling and virtual screening to repurpose and
              develop drug against parasite Leishmania sp. Besides, I also gained hands on experience in the process of crystallizing
              a drug-target protein of Leishmania sp. named Fructose-1-6-bisphosphatase. Performed experimental work on protein
              crystallography beside computational research. Actively participated in designing and execution of the project. 
              Published papers on motif prediction by making independent collaboration Clinical Bioinformatics group of UCD, Ireland.
              Worked and parasitic disease model : Leishmania sp. Wrote small codes in C/C++ to handling output of comparative genomics
              data. Handled plenty of protein and small molecule structures in-silico for structural bioinformatics. Key areas & expertise: 
              Pharmacophore design (Discovery Studio, Pharmer & Ligand Scount), Docking (GOLD), Protein Modeling (Modeller), Protein purification,
              Drug repurposing, Quantum chemical theory based small molecule energy minimization (GAMESS).</td>
          <td>
            2012 to 2014
          </td>
        </tr>




        <tr>
          <td>
            PhD Student at <strong><a href="http://www.jaduniv.edu.in/">Physics Department, Jadavpur University</a></strong>, Kolkata, India.
          </td>
          <td>
            I worked on developing algorithm for numerical characterization and phylogenetic analysis of proteins and peptides. Besides,
             I was also involved with characterizing druggable surface region of pathogen's target proteins, epitope discovery and peptide 
             vaccine design. Performed extensive collaboration with theoritical physicists to develop algorithm. Worked on disease 
             moedel/protein: Influenza virus & Rota virus. Worked & gained lot of experiences on genome, mutational landscape and drug
             design field on Influenza Virus. Also did collaboration with interanational researcher and wet-lab experimental researcher
             to develop idea and published papers. Published papers in corresponding authors and wrote an invited book chapter. Gained
             lots of teaching experiences by taking class in three different colleges as guest lecturer. Key areas & expertise: Sequence Alignment,
             Epitope prediction, Vaccine discovery approaches, Microsoft Excel, Phylogenetic tree drawing algorithms, Protein modelling,
             Mutation analyses .
          <td>
            2007 to 2012
          </td>
        </tr>


      </table>
    </div>
  </section>









  <section id="pubsListBlock">
    <div class="container-fluid">
      <h2>Publications</h2>

      <div class="container-fluid">
        
        <p>Ghosh, A., et al. (2009). &quot;Computational study of dispersion and extent of mutated and duplicated sequences of the H5N1 influenza neuraminidase over the period 1997− 2008.&quot; <u>Journal of chemical information and modeling</u> <b>49</b>(11): 2627-2638. <a href="https://pubs.acs.org/doi/abs/10.1021/ci9001662">link</a> </p>
        <p>Nandy, A., et al. (2009). &quot;Numerical characterization of protein sequences and application to voltage-gated sodium channel α subunit phylogeny.&quot; <u>In silico biology</u> <b>9</b>(3): 77-87. <a href="https://content.iospress.com/articles/in-silico-biology/isb00389">link</a> </p>
        <p>Ghosh, A., et al. (2010). &quot;A STUDY ON PORE FORMATION DYNAMICS IN BIO-MIMICKING MEMBRANE SYSTEM.&quot; <u>Romanian Journal of Biophysics</u> <b>20</b>(4): 347-354. <a href="https://www.rjb.ro/articles/292/Agosh.pdf">link</a> </p>
        <p>Ghosh, A., et al. (2010). &quot;Computational analysis and determination of a highly conserved surface exposed segment in H5N1 avian flu and H1N1 swine flu neuraminidase.&quot; <u>BMC Structural Biology</u> <b>10</b>(1): 6. <a href="https://bmcstructbiol.biomedcentral.com/articles/10.1186/1472-6807-10-6">link</a> </p>
        <p>GHOSH, A. and A. NANDY (2011). &quot;Graphical representation and mathematical characterization of protein sequences and applications to viral proteins.&quot; <u>Advances in protein chemistry and structural biology</u> <b>83</b>: 1. <a href="https://www.sciencedirect.com/science/article/pii/B978012381262900001X">link</a> </p>
        <p>Ghosh, A., et al. (2012). &quot;In Silico Study of Rotavirus VP7 Surface Accessible Conserved Regions for Antiviral Drug/Vaccine Design.&quot; <u>PLoS ONE</u> <b>7</b>(7): e40749. <a href="https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0040749">link</a> </p>
        <p>Ruhanen, H., et al. (2014). &quot;Potential of known and short prokaryotic protein motifs as a basis for novel peptide-based antibacterial therapeutics: a computational survey.&quot; <u>Frontiers in Microbiology</u> <b>5</b>: 4. <a href="https://www.frontiersin.org/articles/10.3389/fmicb.2014.00004/full?utm_source=newsletter&utm_medium=web&utm_campaign=Microbiology-w6-2014">link</a> </p>
        <p>Waugh, B., et al. (2014). &quot;In silico work flow for scaffold hopping in Leishmania.&quot; <u>BMC research notes</u> <b>7</b>(1): 802. <a href="https://link.springer.com/article/10.1186/1756-0500-7-802">link</a> </p>
        <p>Hur, J., et al. (2016). &quot;Design of a RANK-Mimetic Peptide Inhibitor of Osteoclastogenesis with Enhanced RANKL-Binding Affinity.&quot; <u>Molecules and Cells</u> <b>2016</b>(39(4)). <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4844938/">link</a> </p>
        <p>Nguyen, T., et al. (2016). &quot;An ubiquitin-binding molecule can work as an inhibitor of ubiquitin processing enzymes and ubiquitin receptors.&quot; <u>Biochemical and biophysical research communications</u> <b>479</b>(1): 33-39. <a href="https://www.sciencedirect.com/science/article/abs/pii/S0006291X16314590">link</a> </p>
        <p>Jung, H., et al. (2017). &quot;The mutational landscape of ocular marginal zone lymphoma identifies frequent alterations in TNFAIP3 followed by mutations in TBL1XR1 and CREBBP.&quot; <u>Oncotarget</u>. <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5370020/">link</a> </p>
        <p>Kim, C. J., et al. (2018). &quot;The Transcription Factor Ets1 Suppresses T Follicular Helper Type 2 Cell Differentiation to Halt the Onset of Systemic Lupus Erythematosus.&quot; <u>Immunity</u> <b>49</b>(6). <a href="https://www.sciencedirect.com/science/article/pii/S1074761318304771">link</a> </p>
        <p>Ravichandran, S., et al. (2018). &quot;Genome-wide analysis of regulatory G-quadruplexes affecting gene expression in human cytomegalovirus.&quot; <u>PLoS Pathogen</u>. <a href="https://journals.plos.org/plospathogens/article?rev=2&id=10.1371/journal.ppat.1007334">link</a> </p>
        <p>Verma, R., et al. (2018). &quot;Cell surface polysaccharides of Bifidobacterium bifidum induce the generation of Foxp3+ regulatory T cells.&quot; <u>Science Immunology</u> <b>3</b>(28). <a href="https://immunology.sciencemag.org/content/3/28/eaat6975?utm_campaign=toc_imm_2018-10-19&et_rid=79676282&et_cid=2438185">link</a> </p>

      </div>


    </div>
  </section>








  <section id="ambarSkillsBlock">

    <div class="container-fluid">

      <h2>Skills</h2>

      <p>
        My skills are focused around theories and knowledge of biology. I did my bachelor and master degree in pure biological
        sciences and from my PhD I started to develop skills in computational biology. Over the years I have gained both programming
        and biological data analysis skills. Let me present my skillls in a better way <a href="skills.html">in this page</a>.</p>

    </div>

  </section>





  <section id="certDetailsBlock">

    <div class="container-fluid">
    
      <h2>Certificates</h2>
    
      <p>In a overview, domains of my certificates covers mainly academic and few industrial credentials. In addition, recently I am developing new
         skills in web application development and software development in computational biology, which encouraged me to do certifications in
         different programming languages & related tools. Some of the important certificates include the areas like: Lean Six Sigma Training,
         Project Management Training, etc. etc. All my certificates are listed or will be added to my <a href="https://www.linkedin.com/in/ambarnil-ghosh/">LinkedIn</a> profile.</p>

    </div>
  
  </section>




</body>



</html>